content/en/blog/_posts/XX-userns-enabled-by-default/index.md

```diff
@@ -0,0 +1,280 @@
+accessing a mount. When combined with user namespaces, it greatly simplifies the
+support for volumes, as you can forget about the host UID/GIDs the user
+namespace is using.
+
+In particular, thanks to idmap mounts we can:
+ * Run each pod with different UIDs/GIDs on the host. This is key for the
+   lateral movement prevention we mentioned earlier.
+ * Share volumes with pods that don't use user namespaces.
+ * Enable/disable user namespaces without needing to chown the pod's volumes.
+
+However, support for idmap mounts in the kernel is not black/white. The support
+is per file-system and support was added over time to different file-systems.
+
+You can check which kernel version added support for each file-system in the
+`mount_setattr` man page, or checking the online version of it [here][mount_setattr].
+
+Most popular file-systems are supported in some kernel version, the notable
+absence that isn't supported yet is NFS.
+
+[mount_setattr]: https://man7.org/linux/man-pages/man2/mount_setattr.2.html#NOTES
+
+**3. Can you clarify exactly which file-systems need to support idmap mounts?**
+
+The file-systems that need to support idmap mounts are all the file-systems used
+by a pod in the `pod.spec.volumes` field.
+
+This means: for PV/PVC volumes, the file-system used in the PV needs to support
+idmap mounts; for hostPath volumes, the file-system used in the hostPath
+needs to support idmap mounts.
+
+What does this mean for secrets/configmaps/projected/downwardAPI volumes? For
+these volumes, the kubelet creates a `tmpfs` file-system. So, you will need a
+6.3 kernel to use these volumess (note that if you use them as env variables it
+is fine).
+
+And what about emptyDir volumes? Those volumes are created by the Kubelet by
+default in `/var/lib/kubelet/pods/`. You can also use a custom directory for
+this. But what needs to support idmap mounts is the file-system used in that
+directory.
+
+The kubelet creates some more files for the container, like `/etc/hostname`,
+`/etc/resolv.conf`, `/dev/termination-log`, `/etc/hosts`, etc. These files are
+also created in `/var/lib/kubelet/pods/` by default, so it's important for the
+file-system used in that directory to support idmap mounts.
+
+Also, some container runtimes may put some of these ephemeral volumes inside a
+`tmpfs` file-system, in which case you will need support for idmap mounts in
+`tmpfs`.
+
+**4. Can I use a kernel older than 6.3?**
+
+Yes, but you will need to make sure you are not using a `tmpfs` file-system. If
+you avoid that, you can easily use 5.19 (if all the other file-systems you use
+support idmap mounts in that kernel).
+
+To avoid using `tmpfs` can be tricky, though, as we just mentioned. But besides
+having to avoid those volume types, you will also have to avoid mounting the
+service account token. Every pod have it mounted by default, and it is using a
+projected volume that, as we mentioned, uses a `tmpfs` file-system.
```

Comment:
Some grammar fix, as well as slight rewording

```suggestion
It can be tricky to avoid using `tmpfs`, though, as we just described above. 
Besides having to avoid those volume types, you will also have to avoid mounting the
service account token. Every pod has it mounted by default, and it uses a
projected volume that, as we mentioned, uses a `tmpfs` file-system.
```